### DISCUSSION TITLE
How does Talos save dmesg log for previous shutdown

### DISCUSSION THREAD

**Discussion by yingwana3** [siderolabs/talos#7680] [Q&A] [answered]
Hi

When I run workload on talos, some unkown reasons cause the system broken.

For the message shown in the following screen very quickly roll down very quickly, I can't capture the useful information to debug.

<img width="727" alt="image" src="https://github.com/siderolabs/talos/assets/60091241/573af82a-2e66-41bf-b04b-b8df75dbe450">

Can you guys share some methods on how to save the dmesg logs for previous shutdown?

Thanks

**Comment by smira** [comment, marked as answer]
The only way is to send logs out to some other device: https://www.talos.dev/v1.5/talos-guides/configuration/logging/#kernel-logs
